src/plays/digital-delight/components/CategoryFact.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import Math from '../components/images/math.png';
+import Year from '../components/images/year.png';
+import Date from '../components/images/date.png';
+import './Categoryfact.css';
+
+function CategoryFact() {
+  const [fact, setFact] = useState('Click on buttons below to get facts by category');
```

Comment:
Move this to constant
```
'Click on buttons below to get facts by category'
```

```suggestion
  const [fact, setFact] = useState('Click on buttons below to get facts by category');
```